react/ProductSummaryList.js

```diff
@@ -53,7 +53,7 @@ function ProductSummaryList(props) {
     category = '',
     collection,
     hideUnavailableItems = false,
-    orderBy = ORDER_BY_OPTIONS.TOP_SALE_DESC.value,
+    orderBy = ORDER_BY_OPTIONS.DEFAULT.value,
```

Comment:
Shouldn't it be RELEVANCE?
```suggestion
    orderBy = ORDER_BY_OPTIONS.RELEVANCE.value,
```

Reply:
Yeeees!!